### PR TITLE
Book club write ups

### PR DESCRIPTION
**Pull request by StarfallProjects**
Contributions from myself, Will and Ioana

### CODE REVIEW

**Comment by StarfallProjects** [comment]
Can the reviewer (@plaindocs ?) check how the thing actually looks, as still haven't got this building locally?

**Comment by plaindocs** [comment]
If you resolve the merge conflict, you can preview on netlify automatically. Lemme :eyes:

**Comment by plaindocs** [comment]
The conflict is 

```
<<<<<<< bookClubWriteUps
Book Club Write-ups
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The #learn-tech-writing channel in the Write the Docs :doc:`/slack` hosts a book club. This section contains summaries of discussions.

You can check out the :doc:`/book-club/index` here.
=======
Reading List
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
A `list of books </books/>`_ related to technical writing, information architecture, and content strategy.
>>>>>>> master
```

how about something like

```

Books
=====

Book Club Write-ups
~~~~~~~~~~~~~~~~

The #learn-tech-writing channel in the Write the Docs :doc:`/slack` hosts a book club. This section contains summaries of discussions.

You can check out the :doc:`/book-club/index` here.

Reading List
~~~~~~~~~~
A `list of books </books/>`_ related to technical writing, information architecture, and content strategy.

```

**Comment by plaindocs** [comment]
Preview:  https://deploy-preview-1174--dentist-canary-64055.netlify.com/book-club/

**Comment by plaindocs** [comment]
I've added a few links here and there, but there are some issues with the bullet lists and link formatting still. Let me know if you need help with those.

File: docs/book-club/index.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
+========================
+Write the Docs Book Club
+========================
+
+Join us on the Write the Docs :doc:`/slack` in the #learn-tech-writing channel.
+
```
Comment (plaindocs) [inline docs/book-club/index.rst:6]:
```suggestion

.. contents::
   :local:
   :depth: 1
   :backlinks: none
   
```

**Review by plaindocs — APPROVED** [review]
Looks good!

**Pull request merged**